tools/jit/gen_jit_dispatch.py

```diff
@@ -68,10 +68,11 @@ def is_tensor_arg(arg):
         arguments = decl['arguments']
         name = decl['name']
         scalar_args = [arg for arg in arguments if not is_tensor_arg(arg)]
+        has_tensorlist = any(arg['simple_type'] == 'TensorList' for arg in arguments)
 
         # Descriptor is a unique identified for a particular overload of an op
         attr_names = sorted([arg['name'] for arg in scalar_args])
-        num_inputs = len(arguments) - len(scalar_args)
+        num_inputs = len(arguments) - len(scalar_args) if not has_tensorlist else "*"
```

Comment:
I heard from @colesbury that we might be growing methods which take both a `Tensor` and a `TensorList`. Something to keep in mind, though maybe not for this PR.

Reply:
I know, and I thought about fixing it properly, but wanted to the the LSTM running, and I decided to keep things simple. We have an issue open with a list of those ops.